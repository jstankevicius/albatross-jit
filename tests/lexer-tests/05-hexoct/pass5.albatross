04000	02223	03643	0527 070770
06206	00012	00566	0664 004141
04232	07216	07050	0010 027261
005	05750	04545	05252227 02
01041	07601	04550	0070 012250
05527	05355	02731	0036 020326
04240	00051	05614	0443 062227
04001	001	06277	02760017 05
00611	07100	02320	0035 035406
04006	01003	05260	0300 02060
01065	06200	01077	0540 010102
06760	0206	06135	0074 06600
02305	02353	05143	0104 046500
02661	05505	00020	0542 062241
01761	0070	04077	0437 034614
00000	01200	0064	0701 017002
00425	01060	00601	0532 003116
01140	01003	01776	0600 033012
04715	03473	04464	0501 054674
04047	02045	07403	0001 063404
07262	00704	00045	0761 000425
03156	00135	03602	0535 001601
04341	02674	01013	0074 0 000
04661	00124	02003	0100 04001
0605	07611	02475	0524 040404
03246	0264	07320	0521 021504
05270	07054	05211	0070 061203
04227	02204	03000	0754 034401
01710	07670	00045	0424 030004
05074	00042	06401	0057 003
05030	05437	00300	0550 0503
07140	04365	00306	0160 0000
00070	01511	03031	0137 0200
07706	03470	01640	0175 0500
01176	0070	02003	0335 0237
01502	01400	0034	0404 0511
04000	07576	01403	0560 0063
07631	02542	02503	0021 0646
01047	02006	04424	0114 0174
05015	02704	01302	0555 0404
03676	01064	06633	0620 0246
05506	02205	03767	0115 000
07660	00042	01571	0351 0307
00602	05614	0706	0740 0502
06303	06503	0406	0766 060
02102	00072	074	0315426
06014	02503	06511	0021 0474
00610	01661	01407	0450 0362
02017	00000	01402	0502 033
00605	07010	06006	0440 0574
01560	03200	02405	0063 0667
00046	00004	07677	0374 0457
0651	05040	01077	0650 054
0730	0563	00103	0766 0602
05066	02206	07004	0340 0002
02174	04004	01301	0070 0450
06270	00500	0502	0554 0300
05673	01661	07554	0055 0120
01647	03334	00602	0545 0023
01206	07057	00662	0261 0014
06054	05132	04632	0325 0054
01503	05100	04636	0020 0104
0052	0755	01044	0413 0236
01330	07406	00072	0550 0000
0377	04162	07022	0034 0705
01005	07021	00334	0024 0352
06014	02071	03447	0661 000
05326	02300	02340	0042 0406
06507	06010	06052	0672 0000
00006	06462	02270	0760 000
0510	01010	07133	0751 0207
06730	00705	06746	0220 0103
03450	05053	06727	0363 0367
01005	06460	01146	0354 0001
07301	03047	03465	0320 0030
0333	00106	04246	0306 0030
03556	01057	05005	0701 0136
05603	0040	07024	0633 0020
04401	02006	04004	0107 0040
00440	00240	04204	0070 0052
04020	0560	0027	0457 0673
07001	0571	03407	0573 000
0177	02500	04106	0514 057
02303	05040	03700	0604 0335
01606	02201	00415	0007 0425
07214	02002	07070	0230 0562
03030	07320	07055	0060 0074
00614	02010	00504	0007 0214
07700	0730	03410	0623 0455
00760	07445	05500	0203 0300
03043	06600	00255	0003 0770
05006	01007	06010	0634 01
01600	05307	00411	0055 0370
05377	02123	00052	0642 0200
04061	04700	00323	0322 0743
00000	0373	07566	0254 0515
01002	02246	00063	0575 0064
07134	03322	00037	0740 0017
03204	07175	00203	0063 0106
01705	06031	07500	0476 0540
05505	06556	01605	0307 0337
00507	07552	07320	0045 0516
04717	05254	07367	0601 0403
03171	06306	04245	0470 0654
00221	01715	05471	0032 0550
00721	04325	05233	0702 04
03501	00061	01104	0726 0062
00004	07323	07035	0266 0001
00640	06553	00172	0771 0022
00724	02765	02406	0002 0041
02104	07106	00743	0342 0257
00373	00030	04003	0436 0510
02005	05604	01146	0637 0037
02006	03302	06756	0606 0077
00030	03372	06041	0620 0752
0232	03660	00607	0560 0151
06262	04360	01742	0115 0252
05431	00705	06016	0470 0600
03500	06342	00014	0405 0433
01306	05734	07333	0607 0062
02206	02000	01024	0006 0311
07430	00003	00170	0762 0340
01050	03066	06004	0634 0502
02150	06600	01400	0406 000
06503	00710	05267	0045 0243
03177	06366	01637	0413 0603
07401	0340	02004	0726 0007
00350	03100	00140	0553 0202
03033	06011	06033	0064 0050
01147	00046	03521	0431 0053
01104	06177	01632	0777 0600
05614	00433	00012	0360 0425
03344	04513	00100	0641 0556
03646	0546	0222	0463 0124
0020	07604	00026	0276 0703
01500	0055	06625	0307 0012
0501	05401	01644	0566 0310
0410	06406	04720	0104 0041
04000	0263	04202	0006 0270
05343	02350	06501	0200 0303
03133	06602	06016	0450 0510
05002	07003	00221	0516 003
01146	07604	05030	0647 0670
06352	01405	02600	0020 0604
06402	05132	04302	0247 0010
05035	06306	0414	0207 000
00646	01427	07230	0104 073
04012	01736	02537	0044 0246
03070	07100	01447	0746 0402
00010	06504	00070	0703 0370
05006	03020	00260	0022 0014
02552	05065	04355	0004 0066
00347	01337	0163	0006 0002
00202	06064	04306	0500 0272
0706	04210	0166	0606 0001
00504	06103	04605	0003 0164
00011	01620	02340	0302 0030
00702	04602	03320	0624 0104
0430	00636	04000	0103 0123
03001	00067	07005	0707 0760
02066	07050	06006	0447 0007
04730	0202	03706	0522 0004
05512	05060	06340	0260 0051
03301	04406	00500	0040 0272
00013	0025	04503	0702 0756
00400	00314	07531	0402 016
05101	0106	04703	0266 0433
04752	07461	07474	0475 0450
03007	06525	06006	0411 0604
00204	07607	00305	0605 0020
00515	07636	07470	0550 0041
00517	06740	01630	0032 0401
0231	02000	07506	0627 066
01406	0514	07002	0716 0034
00736	0100	00007	0445 0001
01065	00155	04710	0242 0755
07271	0320	00146	0034 0313
00333	04031	06410	0634 0305
07604	06601	0750	0407 0374
0373	03610	01105	0350 0360
07311	0646	06010	0030 0704
00723	05007	03672	0717 0740
05434	05040	0100	0010 0701
03061	05074	01470	0032 0147
00165	00040	03206	0506 0247
04057	01500	03051	0034 0450
00705	04011	03507	0067 0012
05602	01002	06020	0037 0100
007	02164	06576	0031263
00007	00731	06612	0006 0555
0175	005	04450	003300
06172	07503	00072	0305 0745
00172	05360	00503	0362 0300
02714	01144	03330	0050 0076
05304	00053	04643	0057 03
00403	00065	07500	0652 0510
02111	03303	05400	0551 0002
04300	01400	0524	0405 0356
00043	07720	01111	0002 0044
01560	06017	01031	0702 0270
