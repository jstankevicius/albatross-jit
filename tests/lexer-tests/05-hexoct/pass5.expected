1 1 INT 2048
7 1 INT 1171
13 1 INT 1955
19 1 INT 343
24 1 INT 29176
1 2 INT 3206
7 2 INT 10
13 2 INT 374
19 2 INT 436
24 2 INT 2145
1 3 INT 2202
7 3 INT 3726
13 3 INT 3624
19 3 INT 8
24 3 INT 11953
1 4 INT 5
5 4 INT 3048
11 4 INT 2405
17 4 INT 1397911
26 4 INT 2
1 5 INT 545
7 5 INT 3969
13 5 INT 2408
19 5 INT 56
24 5 INT 5288
1 6 INT 2903
7 6 INT 2797
13 6 INT 1497
19 6 INT 30
24 6 INT 8406
1 7 INT 2208
7 7 INT 41
13 7 INT 2956
19 7 INT 291
24 7 INT 25751
1 8 INT 2049
7 8 INT 1
11 8 INT 3263
17 8 INT 778255
26 8 INT 5
1 9 INT 393
7 9 INT 3648
13 9 INT 1232
19 9 INT 29
24 9 INT 15110
1 10 INT 2054
7 10 INT 515
13 10 INT 2736
19 10 INT 192
24 10 INT 1072
1 11 INT 565
7 11 INT 3200
13 11 INT 575
19 11 INT 352
24 11 INT 4162
1 12 INT 3568
7 12 INT 134
12 12 INT 3165
18 12 INT 60
23 12 INT 3456
1 13 INT 1221
7 13 INT 1259
13 13 INT 2659
19 13 INT 68
24 13 INT 19776
1 14 INT 1457
7 14 INT 2885
13 14 INT 16
19 14 INT 354
24 14 INT 25761
1 15 INT 1009
7 15 INT 56
12 15 INT 2111
18 15 INT 287
23 15 INT 14732
1 16 INT 0
7 16 INT 640
13 16 INT 52
18 16 INT 449
23 16 INT 7682
1 17 INT 277
7 17 INT 560
13 17 INT 385
19 17 INT 346
24 17 INT 1614
1 18 INT 608
7 18 INT 515
13 18 INT 1022
19 18 INT 384
24 18 INT 13834
1 19 INT 2509
7 19 INT 1851
13 19 INT 2356
19 19 INT 321
24 19 INT 22972
1 20 INT 2087
7 20 INT 1061
13 20 INT 3843
19 20 INT 1
24 20 INT 26372
1 21 INT 3762
7 21 INT 452
13 21 INT 37
19 21 INT 497
24 21 INT 277
1 22 INT 1646
7 22 INT 93
13 22 INT 1922
19 22 INT 349
24 22 INT 897
1 23 INT 2273
7 23 INT 1468
13 23 INT 523
19 23 INT 60
24 23 INT 0
26 23 INT 0
1 24 INT 2481
7 24 INT 84
13 24 INT 1027
19 24 INT 64
24 24 INT 2049
1 25 INT 389
6 25 INT 3977
12 25 INT 1341
18 25 INT 340
23 25 INT 16644
1 26 INT 1702
7 26 INT 180
12 26 INT 3792
18 26 INT 337
23 26 INT 9028
1 27 INT 2744
7 27 INT 3628
13 27 INT 2697
19 27 INT 56
24 27 INT 25219
1 28 INT 2199
7 28 INT 1156
13 28 INT 1536
19 28 INT 492
24 28 INT 14593
1 29 INT 968
7 29 INT 4024
13 29 INT 37
19 29 INT 276
24 29 INT 12292
1 30 INT 2620
7 30 INT 34
13 30 INT 3329
19 30 INT 47
24 30 INT 3
1 31 INT 2584
7 31 INT 2847
13 31 INT 192
19 31 INT 360
24 31 INT 323
1 32 INT 3680
7 32 INT 2293
13 32 INT 198
19 32 INT 112
24 32 INT 0
1 33 INT 56
7 33 INT 841
13 33 INT 1561
19 33 INT 95
24 33 INT 128
1 34 INT 4038
7 34 INT 1848
13 34 INT 928
19 34 INT 125
24 34 INT 320
1 35 INT 638
7 35 INT 56
12 35 INT 1027
18 35 INT 221
23 35 INT 159
1 36 INT 834
7 36 INT 768
13 36 INT 28
18 36 INT 260
23 36 INT 329
1 37 INT 2048
7 37 INT 3966
13 37 INT 771
19 37 INT 368
24 37 INT 51
1 38 INT 3993
7 38 INT 1378
13 38 INT 1347
19 38 INT 17
24 38 INT 422
1 39 INT 551
7 39 INT 1030
13 39 INT 2324
19 39 INT 76
24 39 INT 124
1 40 INT 2573
7 40 INT 1476
13 40 INT 706
19 40 INT 365
24 40 INT 260
1 41 INT 1982
7 41 INT 564
13 41 INT 3483
19 41 INT 400
24 41 INT 166
1 42 INT 2886
7 42 INT 1157
13 42 INT 2039
19 42 INT 77
24 42 INT 0
1 43 INT 4016
7 43 INT 34
13 43 INT 889
19 43 INT 233
24 43 INT 199
1 44 INT 386
7 44 INT 2956
13 44 INT 454
18 44 INT 480
23 44 INT 322
1 45 INT 3267
7 45 INT 3395
13 45 INT 262
18 45 INT 502
23 45 INT 48
1 46 INT 1090
7 46 INT 58
13 46 INT 60
17 46 INT 105238
1 47 INT 3084
7 47 INT 1347
13 47 INT 3401
19 47 INT 17
24 47 INT 316
1 48 INT 392
7 48 INT 945
13 48 INT 775
19 48 INT 296
24 48 INT 242
1 49 INT 1039
7 49 INT 0
13 49 INT 770
19 49 INT 322
24 49 INT 27
1 50 INT 389
7 50 INT 3592
13 50 INT 3078
19 50 INT 288
24 50 INT 380
1 51 INT 880
7 51 INT 1664
13 51 INT 1285
19 51 INT 51
24 51 INT 439
1 52 INT 38
7 52 INT 4
13 52 INT 4031
19 52 INT 252
24 52 INT 303
1 53 INT 425
6 53 INT 2592
12 53 INT 575
18 53 INT 424
23 53 INT 44
1 54 INT 472
6 54 INT 371
11 54 INT 67
17 54 INT 502
22 54 INT 386
1 55 INT 2614
7 55 INT 1158
13 55 INT 3588
19 55 INT 224
24 55 INT 2
1 56 INT 1148
7 56 INT 2052
13 56 INT 705
19 56 INT 56
24 56 INT 296
1 57 INT 3256
7 57 INT 320
13 57 INT 322
18 57 INT 364
23 57 INT 192
1 58 INT 3003
7 58 INT 945
13 58 INT 3948
19 58 INT 45
24 58 INT 80
1 59 INT 935
7 59 INT 1756
13 59 INT 386
19 59 INT 357
24 59 INT 19
1 60 INT 646
7 60 INT 3631
13 60 INT 434
19 60 INT 177
24 60 INT 12
1 61 INT 3116
7 61 INT 2650
13 61 INT 2458
19 61 INT 213
24 61 INT 44
1 62 INT 835
7 62 INT 2624
13 62 INT 2462
19 62 INT 16
24 62 INT 68
1 63 INT 42
6 63 INT 493
11 63 INT 548
17 63 INT 267
22 63 INT 158
1 64 INT 728
7 64 INT 3846
13 64 INT 58
19 64 INT 360
24 64 INT 0
1 65 INT 255
6 65 INT 2162
12 65 INT 3602
18 65 INT 28
23 65 INT 453
1 66 INT 517
7 66 INT 3601
13 66 INT 220
19 66 INT 20
24 66 INT 234
1 67 INT 3084
7 67 INT 1081
13 67 INT 1831
19 67 INT 433
24 67 INT 0
1 68 INT 2774
7 68 INT 1216
13 68 INT 1248
19 68 INT 34
24 68 INT 262
1 69 INT 3399
7 69 INT 3080
13 69 INT 3114
19 69 INT 442
24 69 INT 0
1 70 INT 6
7 70 INT 3378
13 70 INT 1208
19 70 INT 496
24 70 INT 0
1 71 INT 328
6 71 INT 520
12 71 INT 3675
18 71 INT 489
23 71 INT 135
1 72 INT 3544
7 72 INT 453
13 72 INT 3558
19 72 INT 144
24 72 INT 67
1 73 INT 1832
7 73 INT 2603
13 73 INT 3543
19 73 INT 243
24 73 INT 247
1 74 INT 517
7 74 INT 3376
13 74 INT 614
19 74 INT 236
24 74 INT 1
1 75 INT 3777
7 75 INT 1575
13 75 INT 1845
19 75 INT 208
24 75 INT 24
1 76 INT 219
6 76 INT 70
12 76 INT 2214
18 76 INT 198
23 76 INT 24
1 77 INT 1902
7 77 INT 559
13 77 INT 2565
19 77 INT 449
24 77 INT 94
1 78 INT 2947
7 78 INT 32
12 78 INT 3604
18 78 INT 411
23 78 INT 16
1 79 INT 2305
7 79 INT 1030
13 79 INT 2052
19 79 INT 71
24 79 INT 32
1 80 INT 288
7 80 INT 160
13 80 INT 2180
19 80 INT 56
24 80 INT 42
1 81 INT 2064
7 81 INT 368
12 81 INT 23
17 81 INT 303
22 81 INT 443
1 82 INT 3585
7 82 INT 377
12 82 INT 1799
18 82 INT 379
23 82 INT 0
1 83 INT 127
6 83 INT 1344
12 83 INT 2118
18 83 INT 332
23 83 INT 47
1 84 INT 1219
7 84 INT 2592
13 84 INT 1984
19 84 INT 388
24 84 INT 221
1 85 INT 902
7 85 INT 1153
13 85 INT 269
19 85 INT 7
24 85 INT 277
1 86 INT 3724
7 86 INT 1026
13 86 INT 3640
19 86 INT 152
24 86 INT 370
1 87 INT 1560
7 87 INT 3792
13 87 INT 3629
19 87 INT 48
24 87 INT 60
1 88 INT 396
7 88 INT 1032
13 88 INT 324
19 88 INT 7
24 88 INT 140
1 89 INT 4032
7 89 INT 472
12 89 INT 1800
18 89 INT 403
23 89 INT 301
1 90 INT 496
7 90 INT 3877
13 90 INT 2880
19 90 INT 131
24 90 INT 192
1 91 INT 1571
7 91 INT 3456
13 91 INT 173
19 91 INT 3
24 91 INT 504
1 92 INT 2566
7 92 INT 519
13 92 INT 3080
19 92 INT 412
24 92 INT 1
1 93 INT 896
7 93 INT 2759
13 93 INT 265
19 93 INT 45
24 93 INT 248
1 94 INT 2815
7 94 INT 1107
13 94 INT 42
19 94 INT 418
24 94 INT 128
1 95 INT 2097
7 95 INT 2496
13 95 INT 211
19 95 INT 210
24 95 INT 483
1 96 INT 0
7 96 INT 251
12 96 INT 3958
18 96 INT 172
23 96 INT 333
1 97 INT 514
7 97 INT 1190
13 97 INT 51
19 97 INT 381
24 97 INT 52
1 98 INT 3676
7 98 INT 1746
13 98 INT 31
19 98 INT 480
24 98 INT 15
1 99 INT 1668
7 99 INT 3709
13 99 INT 131
19 99 INT 51
24 99 INT 70
1 100 INT 965
7 100 INT 3097
13 100 INT 3904
19 100 INT 318
24 100 INT 352
1 101 INT 2885
7 101 INT 3438
13 101 INT 901
19 101 INT 199
24 101 INT 223
1 102 INT 327
7 102 INT 3946
13 102 INT 3792
19 102 INT 37
24 102 INT 334
1 103 INT 2511
7 103 INT 2732
13 103 INT 3831
19 103 INT 385
24 103 INT 259
1 104 INT 1657
7 104 INT 3270
13 104 INT 2213
19 104 INT 312
24 104 INT 428
1 105 INT 145
7 105 INT 973
13 105 INT 2873
19 105 INT 26
24 105 INT 360
1 106 INT 465
7 106 INT 2261
13 106 INT 2715
19 106 INT 450
24 106 INT 4
1 107 INT 1857
7 107 INT 49
13 107 INT 580
19 107 INT 470
24 107 INT 50
1 108 INT 4
7 108 INT 3795
13 108 INT 3613
19 108 INT 182
24 108 INT 1
1 109 INT 416
7 109 INT 3435
13 109 INT 122
19 109 INT 505
24 109 INT 18
1 110 INT 468
7 110 INT 1525
13 110 INT 1286
19 110 INT 2
24 110 INT 33
1 111 INT 1092
7 111 INT 3654
13 111 INT 483
19 111 INT 226
24 111 INT 175
1 112 INT 251
7 112 INT 24
13 112 INT 2051
19 112 INT 286
24 112 INT 328
1 113 INT 1029
7 113 INT 2948
13 113 INT 614
19 113 INT 415
24 113 INT 31
1 114 INT 1030
7 114 INT 1730
13 114 INT 3566
19 114 INT 390
24 114 INT 63
1 115 INT 24
7 115 INT 1786
13 115 INT 3105
19 115 INT 400
24 115 INT 490
1 116 INT 154
6 116 INT 1968
12 116 INT 391
18 116 INT 368
23 116 INT 105
1 117 INT 3250
7 117 INT 2288
13 117 INT 994
19 117 INT 77
24 117 INT 170
1 118 INT 2841
7 118 INT 453
13 118 INT 3086
19 118 INT 312
24 118 INT 384
1 119 INT 1856
7 119 INT 3298
13 119 INT 12
19 119 INT 261
24 119 INT 283
1 120 INT 710
7 120 INT 3036
13 120 INT 3803
19 120 INT 391
24 120 INT 50
1 121 INT 1158
7 121 INT 1024
13 121 INT 532
19 121 INT 6
24 121 INT 201
1 122 INT 3864
7 122 INT 3
13 122 INT 120
19 122 INT 498
24 122 INT 224
1 123 INT 552
7 123 INT 1590
13 123 INT 3076
19 123 INT 412
24 123 INT 322
1 124 INT 1128
7 124 INT 3456
13 124 INT 768
19 124 INT 262
24 124 INT 0
1 125 INT 3395
7 125 INT 456
13 125 INT 2743
19 125 INT 37
24 125 INT 163
1 126 INT 1663
7 126 INT 3318
13 126 INT 927
19 126 INT 267
24 126 INT 387
1 127 INT 3841
7 127 INT 224
12 127 INT 1028
18 127 INT 470
23 127 INT 7
1 128 INT 232
7 128 INT 1600
13 128 INT 96
19 128 INT 363
24 128 INT 130
1 129 INT 1563
7 129 INT 3081
13 129 INT 3099
19 129 INT 52
24 129 INT 40
1 130 INT 615
7 130 INT 38
13 130 INT 1873
19 130 INT 281
24 130 INT 43
1 131 INT 580
7 131 INT 3199
13 131 INT 922
19 131 INT 511
24 131 INT 384
1 132 INT 2956
7 132 INT 283
13 132 INT 10
19 132 INT 240
24 132 INT 277
1 133 INT 1764
7 133 INT 2379
13 133 INT 64
19 133 INT 417
24 133 INT 366
1 134 INT 1958
7 134 INT 358
12 134 INT 146
17 134 INT 307
22 134 INT 84
1 135 INT 16
6 135 INT 3972
12 135 INT 22
18 135 INT 190
23 135 INT 451
1 136 INT 832
7 136 INT 45
12 136 INT 3477
18 136 INT 199
23 136 INT 10
1 137 INT 321
6 137 INT 2817
12 137 INT 932
18 137 INT 374
23 137 INT 200
1 138 INT 264
6 138 INT 3334
12 138 INT 2512
18 138 INT 68
23 138 INT 33
1 139 INT 2048
7 139 INT 179
12 139 INT 2178
18 139 INT 6
23 139 INT 184
1 140 INT 2787
7 140 INT 1256
13 140 INT 3393
19 140 INT 128
24 140 INT 195
1 141 INT 1627
7 141 INT 3458
13 141 INT 3086
19 141 INT 296
24 141 INT 328
1 142 INT 2562
7 142 INT 3587
13 142 INT 145
19 142 INT 334
24 142 INT 3
1 143 INT 614
7 143 INT 3972
13 143 INT 2584
19 143 INT 423
24 143 INT 440
1 144 INT 3306
7 144 INT 773
13 144 INT 1408
19 144 INT 16
24 144 INT 388
1 145 INT 3330
7 145 INT 2650
13 145 INT 2242
19 145 INT 167
24 145 INT 8
1 146 INT 2589
7 146 INT 3270
13 146 INT 268
18 146 INT 135
23 146 INT 0
1 147 INT 422
7 147 INT 791
13 147 INT 3736
19 147 INT 68
24 147 INT 59
1 148 INT 2058
7 148 INT 990
13 148 INT 1375
19 148 INT 36
24 148 INT 166
1 149 INT 1592
7 149 INT 3648
13 149 INT 807
19 149 INT 486
24 149 INT 258
1 150 INT 8
7 150 INT 3396
13 150 INT 56
19 150 INT 451
24 150 INT 248
1 151 INT 2566
7 151 INT 1552
13 151 INT 176
19 151 INT 18
24 151 INT 12
1 152 INT 1386
7 152 INT 2613
13 152 INT 2285
19 152 INT 4
24 152 INT 54
1 153 INT 231
7 153 INT 735
13 153 INT 115
18 153 INT 6
23 153 INT 2
1 154 INT 130
7 154 INT 3124
13 154 INT 2246
19 154 INT 320
24 154 INT 186
1 155 INT 454
6 155 INT 2184
12 155 INT 118
17 155 INT 390
22 155 INT 1
1 156 INT 324
7 156 INT 3139
13 156 INT 2437
19 156 INT 3
24 156 INT 116
1 157 INT 9
7 157 INT 912
13 157 INT 1248
19 157 INT 194
24 157 INT 24
1 158 INT 450
7 158 INT 2434
13 158 INT 1744
19 158 INT 404
24 158 INT 68
1 159 INT 280
6 159 INT 414
12 159 INT 2048
18 159 INT 67
23 159 INT 83
1 160 INT 1537
7 160 INT 55
13 160 INT 3589
19 160 INT 455
24 160 INT 496
1 161 INT 1078
7 161 INT 3624
13 161 INT 3078
19 161 INT 295
24 161 INT 7
1 162 INT 2520
7 162 INT 130
12 162 INT 1990
18 162 INT 338
23 162 INT 4
1 163 INT 2890
7 163 INT 2608
13 163 INT 3296
19 163 INT 176
24 163 INT 41
1 164 INT 1729
7 164 INT 2310
13 164 INT 320
19 164 INT 32
24 164 INT 186
1 165 INT 11
7 165 INT 21
12 165 INT 2371
18 165 INT 450
23 165 INT 494
1 166 INT 256
7 166 INT 204
13 166 INT 3929
19 166 INT 258
24 166 INT 14
1 167 INT 2625
7 167 INT 70
12 167 INT 2499
18 167 INT 182
23 167 INT 283
1 168 INT 2538
7 168 INT 3889
13 168 INT 3900
19 168 INT 317
24 168 INT 296
1 169 INT 1543
7 169 INT 3413
13 169 INT 3078
19 169 INT 265
24 169 INT 388
1 170 INT 132
7 170 INT 3975
13 170 INT 197
19 170 INT 389
24 170 INT 16
1 171 INT 333
7 171 INT 3998
13 171 INT 3896
19 171 INT 360
24 171 INT 33
1 172 INT 335
7 172 INT 3552
13 172 INT 920
19 172 INT 26
24 172 INT 257
1 173 INT 153
6 173 INT 1024
12 173 INT 3910
18 173 INT 407
23 173 INT 54
1 174 INT 774
7 174 INT 332
12 174 INT 3586
18 174 INT 462
23 174 INT 28
1 175 INT 478
7 175 INT 64
12 175 INT 7
18 175 INT 293
23 175 INT 1
1 176 INT 565
7 176 INT 109
13 176 INT 2504
19 176 INT 162
24 176 INT 493
1 177 INT 3769
7 177 INT 208
12 177 INT 102
18 177 INT 28
23 177 INT 203
1 178 INT 219
7 178 INT 2073
13 178 INT 3336
19 178 INT 412
24 178 INT 197
1 179 INT 3972
7 179 INT 3457
13 179 INT 488
18 179 INT 263
23 179 INT 252
1 180 INT 251
6 180 INT 1928
12 180 INT 581
18 180 INT 232
23 180 INT 240
1 181 INT 3785
7 181 INT 422
12 181 INT 3080
18 181 INT 24
23 181 INT 452
1 182 INT 467
7 182 INT 2567
13 182 INT 1978
19 182 INT 463
24 182 INT 480
1 183 INT 2844
7 183 INT 2592
13 183 INT 64
18 183 INT 8
23 183 INT 449
1 184 INT 1585
7 184 INT 2620
13 184 INT 824
19 184 INT 26
24 184 INT 103
1 185 INT 117
7 185 INT 32
13 185 INT 1670
19 185 INT 326
24 185 INT 167
1 186 INT 2095
7 186 INT 832
13 186 INT 1577
19 186 INT 28
24 186 INT 296
1 187 INT 453
7 187 INT 2057
13 187 INT 1863
19 187 INT 55
24 187 INT 10
1 188 INT 2946
7 188 INT 514
13 188 INT 3088
19 188 INT 31
24 188 INT 64
1 189 INT 7
5 189 INT 1140
11 189 INT 3454
17 189 INT 12979
1 190 INT 7
7 190 INT 473
13 190 INT 3466
19 190 INT 6
24 190 INT 365
1 191 INT 125
6 191 INT 5
10 191 INT 2344
16 191 INT 1728
1 192 INT 3194
7 192 INT 3907
13 192 INT 58
19 192 INT 197
24 192 INT 485
1 193 INT 122
7 193 INT 2800
13 193 INT 323
19 193 INT 242
24 193 INT 192
1 194 INT 1484
7 194 INT 612
13 194 INT 1752
19 194 INT 40
24 194 INT 62
1 195 INT 2756
7 195 INT 43
13 195 INT 2467
19 195 INT 47
24 195 INT 3
1 196 INT 259
7 196 INT 53
13 196 INT 3904
19 196 INT 426
24 196 INT 328
1 197 INT 1097
7 197 INT 1731
13 197 INT 2816
19 197 INT 361
24 197 INT 2
1 198 INT 2240
7 198 INT 768
13 198 INT 340
18 198 INT 261
23 198 INT 238
1 199 INT 35
7 199 INT 4048
13 199 INT 585
19 199 INT 2
24 199 INT 36
1 200 INT 880
7 200 INT 3087
13 200 INT 537
19 200 INT 450
24 200 INT 184
